src/cosmology/compat/camb/__init__.py

```diff
@@ -54,6 +54,11 @@ def hubble_distance(self) -> Array:
         """Hubble distance."""
         return np.array(299792.458 / self.pars.H0)
 
+    @property
+    def critical_density0(self) -> Array:
+        r"""Critical density today :math:`3*H_0^2/(8*\pi*G)`."""
+        return np.array(3 * self.pars.H0**2 / (8 * np.pi * 4.3009172706e-9))
+
```

Comment:
Nicely worked out! Nevertheless, I would prefer to use what CAMB reports as its critical density, to prevent inconsistency issues
```suggestion
    @property
    def critical_density0(self) -> Array:
        """Critical density today in Msol Mpc-3."""
        # gravitational constant kappa = 8pi G/c^2 in Mpc Msol-1
        # uses nominal value of (G Msol) following IAU 2015
        KAPPA = 1.202706180375887e-18
        return np.array(self.results.grhocrit / KAPPA)
```

Reply:
I did see `grhocrit` but wasn't entirely sure what it was...

Reply:
Yeah, after reverse engineering, turns out it's $G \cdot \rho_{\rm crit}$